src/main/scala/is/hail/expr/ir/functions/Functions.scala

```diff
@@ -43,7 +43,15 @@ object IRFunctionRegistry {
         (ts, t2s).zipped.forall(_.unify(_))
       }
     }
-    val validIR = lookupInRegistry[Conversion](irRegistry, name, args, findIR).map(_._2)
+    val validIR = lookupInRegistry[Conversion](irRegistry, name, args, findIR).map {
+      case (_, conversion) =>
+        { irs: Seq[IR] =>
+          if (args.forall(!_.isInstanceOf[TAggregable]))
+            ApplyIR(name, irs, conversion)
+          else
+            conversion(irs)
```

Comment:
Is there a low-level IR test that checks that this works when there is an aggregator (i.e. a test that would fail if you didn't make this check)?

Reply:
most of the aggregator stuff would fail (and did fail) because the aggregators don't get extracted in the right spot. I'm not quite sure if that's what your asking?

Reply:
is it easy to make a test in FunctionsSuite that would test this? I'm wondering if this is caught by a test on the lower-level scala side, rather than a python-api test.

Reply:
tests that call aggregators that are stored in the function registry will fail (GroupBySuite, etc.) I can add an aggregator test to the FunctionSuite as well.

Reply:
ok, i added a test